base-integration/src/utilities/helper.ts

```diff
@@ -6,6 +6,10 @@ export class Helper {
 
     let finalData = data.trim();
 
+    finalData = finalData.replace('\r\n', ' ')
+    finalData = finalData.replace('\r', ' ')
+    finalData = finalData.replace('\n', ' ')
+
```

Comment:
```suggestion
    const regex = /\/r|\/n/g
    finalData = finalData.replace(regex, ' ')
```

Reply:
@schwamman we should prob. keep the original format, that way `\r\n` doesn't create two spaces.
```suggestion
    finalData = finalData.replace(/\r\n/g, ' ')
    finalData = finalData.replace(/\r/g, ' ')
    finalData = finalData.replace(/\n/g, ' ')

```